.github/workflows/unit.yml

```
@@ -63,17 +74,21 @@ jobs:
       - run: cargo nextest run --release -p ef-tests --features ef-tests
 
   doc:
-    name: doc tests
+    name: doc tests (${{ matrix.network }})
     runs-on:
       group: Reth
     timeout-minutes: 30
+    strategy:
+      matrix:
+        network: ["ethereum", "optimism"]
     steps:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
       - uses: Swatinem/rust-cache@v2
         with:
           cache-on-failure: true
-      - run: cargo test --doc --workspace --all-features
+      - name: Run doctests
+        run: cargo test --doc --all --features "jemalloc-prof,${{ matrix.network }}"
```

We can remove jemalloc from ci, I think "--all-features" was unnecessary

```suggestion
        run: cargo test --doc --workspace --features "${{ matrix.network }}"
```